2018-03-05 09:07:22,519 tools_module.py[line:227] ERROR app token验证失败: api=/api/add_driving_data, args=None, auth_token=196bca8c5aa54224b8c2b796f5664239, token=None, headers=Accept-Encoding: gzip
Auth-Token: 196bca8c5aa54224b8c2b796f5664239
User-Agent: okhttp/3.9.1
Connection: Keep-Alive
Host: safego.org:5000
Content-Length: 15077
Cache-Control: no-cache
Content-Type: multipart/form-data; boundary=743412ec-6c9d-4611-bb06-fb50245057ca

NoneType
